src/components/Transactions/Transfer/TransferAddModal.js

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { get, sortBy } from 'lodash';
+
+import { useShowToast } from '@folio/stripes-acq-components';
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+
+import TransferAddModalForm from './TransferAddForm';
+import {
+  budgetResource,
+  fundsResource,
+  transactionResource,
+} from '../../../common/resources';
+
+const TransferAddModal = ({
+  budgetName,
+  fiscalYearId,
+  mutator,
+  onClose,
+  resources,
+  stripes,
+}) => {
+  const showCallout = useShowToast();
+
+  const saveTransfer = useCallback(
+    async (formValue) => {
+      try {
+        const transfer = await mutator.transaction.POST({
+          ...formValue,
+          fiscalYearId,
+          transactionType: 'Transfer',
+          currency: stripes.currency,
+          source: 'User'
+        });
+        const { amount } = transfer;
+        showCallout('ui-finance.transactions.transfer.hasBeenCreated', 'success', { amount, budgetName });
+        onClose();
+      } catch (e) {
+        showCallout('ui-finance.transactions.transfer.hasNotBeenCreated', 'error', { amount: formValue.amount, budgetName });
+      }
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [budgetName, fiscalYearId, mutator]
+  );
+
+  const funds = sortBy(get(resources, ['funds', 'records'], []).map(f => ({ label: f.name, value: f.id })), 'label');
```

Comment:
I don't think we should sort values on ui. This should be done at the back-end, please fix corresponding back-end queries